From a1e5f1614710541284fc2b32b399e49814e343eb Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Mon, 16 Oct 2017 11:53:18 +1100
Subject: [PATCH] Solus tweaks to the default config

 - Use solus icon for kickoff
 - Remove toolbox icon from desktop

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 applets/kickoff/package/contents/config/main.xml      | 2 +-
 containments/desktop/package/contents/config/main.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/applets/kickoff/package/contents/config/main.xml b/applets/kickoff/package/contents/config/main.xml
index be031dc..3806910 100644
--- a/applets/kickoff/package/contents/config/main.xml
+++ b/applets/kickoff/package/contents/config/main.xml
@@ -16,7 +16,7 @@
     </entry>
     <entry name="icon" type="String">
       <label>The name of the icon used in the compact representation (e.g. on a small panel).</label>
-      <default>start-here-kde</default>
+      <default>start-here-solus</default>
     </entry>
     <entry name="favorites" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
diff --git a/containments/desktop/package/contents/config/main.xml b/containments/desktop/package/contents/config/main.xml
index 6830fb0..cfe64fc 100644
--- a/containments/desktop/package/contents/config/main.xml
+++ b/containments/desktop/package/contents/config/main.xml
@@ -134,7 +134,7 @@
     </entry>
     <entry name="showToolbox" type="Bool">
       <label>Whether to show the Desktop Toolbox. Only used when serving as containment.</label>
-      <default>true</default>
+      <default>false</default>
     </entry>
     <entry name="pressToMove" type="Bool">
       <label>Whether to activate widget handles and widget move mode by long press. Only used when serving as containment.</label>
-- 
2.14.2

